pkgs/development/python-modules/gmqtt/default.nix

```diff
@@ -0,0 +1,41 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+
+  # build-system
+  setuptools,
+
+  # tests
+  pytestCheckHook,
+  pytest-asyncio,
+}:
+
+buildPythonPackage rec {
+  pname = "gmqtt";
+  version = "0.6.16";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-3dH9wcauYE50N3z3DpnwZ+V5wDwccaas1JThmek7f6Q=";
```

Comment:
```suggestion
    hash = "sha256-3dH9wcauYE50N3z3DpnwZ+V5wDwccaas1JThmek7f6Q=";
```
nitpick

Reply:
> nitpick

Thanks (blaming the `nix-init`'s templates). Fixed.